groovy-language-server/src/test/java/com/palantir/ls/groovy/compilation/GroovycWrapperTest.java

```diff
@@ -174,6 +174,27 @@ public void testCompile() throws InterruptedException, ExecutionException, IOExc
     }
 
     @Test
+    public void testCompile_WithJavaExtension() throws InterruptedException, ExecutionException, IOException {
+        addFileToFolder(root.getRoot(), "test1.groovy",
+                "class Coordinates {\n"
+                        + "   double latitude\n"
+                        + "   double longitude\n"
+                        + "   double getAt(int idx) {\n"
+                        + "      if (idx == 0) latitude\n"
+                        + "      else if (idx == 1) longitude\n"
+                        + "      else throw new ExceptionNew(\"Wrong coordinate index, use 0 or 1\")\n"
+                        + "   }\n"
+                        + "}\n");
+        addFileToFolder(root.getRoot(), "ExceptionNew.java", "public class ExceptionNew {}");
```

Comment:
let's add some extra tests:
1. inside the java file only define a new class ExceptionOld
2. only have a java file where the syntax will fail even when evaluating with groovy syntax